### PR TITLE
Add about String Iterator fix

### PR DESCRIPTION
**Pull request by SandaruJayawardana**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-beta2-release-note.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 To view bug fixes, see the [GitHub milestone for Swan Lake Beta2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta2%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
 
+Set the value type of the string iteration sequence to string:Char.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta2-release-note.md:43]: Let's move this to the "Improvements" section.

It is only the type that changed, right? Shall we say

```suggestion
- The static type of string iteration has been changed from `string` to `string:Char`.
```

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-beta2-release-note.md (hunk 1 above)
Comment (SandaruJayawardana) [reply release-notes/swan-lake-beta2-release-note.md:43]: Yes, I'll change it.

File: release-notes/swan-lake-beta2-release-note.md [hunk 2]
```diff
@@ -36,6 +36,16 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### Improvements
 
+The static type of string iteration has been changed from `string` to `string:Char`.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta2-release-note.md:39]:
```suggestion
- The static type of string iteration has been changed from `string` to `string:Char`.
```

File: release-notes/swan-lake-beta2-release-note.md [hunk 3]
```diff
@@ -36,6 +36,16 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
...
+
+public function main() {
+    string str = "foo";
+
+    foreach string:Char s in str {
+        
+    }
+}
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta2-release-note.md:47]:
````suggestion
```ballerina
public function main() {
    string str = "foo";

    foreach string:Char s in str {
        io:println(s);        
    }

    record {| string:Char value; |}? next = str.iterator().next();

    if !(next is ()) {
        io:println(next.value);
    }
}
```
````

**Review by gimantha — APPROVED** [review]

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**